### ISSUE TITLE
Error: pngquant failed to build, make sure that libpng-dev is installed

### ISSUE THREAD

**Issue by jgpasch** [react-boilerplate/react-boilerplate#2513]
Before opening a new issue, please take a moment to review our [**community guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md) to make the contribution process easy and effective for everyone involved.

## Description
    npm run setup 
is failing on my windows 7 machine. The error says:

    pngquant failed to build, make sure that libpng-dev is installed

## Steps to reproduce
1. clone the repository with the steps provided on the homepage
2. run the command "npm run setup"

**Expected behavior**
"npm run setup" is expected to install dependencies and clean the git repo.

**Screenshots**
![image](https://user-images.githubusercontent.com/7545880/49604618-5d27d880-f95c-11e8-8c28-2c73241e5aef.png)

## Versions

- React-Boilerplate: 3.7.0
- Node/NPM:  10.13.0 / 6.4.1
- Browser: N/A

**Comment by gabrycaos** [comment]
libpng is an OS-native library, you have to have it installed on your system before start the setup

**Comment by julienben** [comment]
I don't think that's true for the majority of setups. Most of the time the install process should be able to take care of this by itself (like with node-sass and libsass). It's not surprising that it would be buggy in some setups though (especially Windows ones). You should indeed try installing it in your system manually before continuing with the react-boilerplate install.

Also, did you google this issue before posting? It's a fairly common problem.

Will close this as there's nothing we can do about it here IMO.

**Comment by jgpasch** [comment]
Thank you both for your comments. 

Yes of course I googled the issue before I posted this. I figured this was a common issue for other people using Windows (I have to use it for work), and thought it would be helpful for people running into the same issue in the future. I saw someone had opened an issue about this same thing before, but they did not follow the proper issue format and the thread was closed.

I ended up just removing the image optimization functionality which also took care of the problem.
